README.md

```diff
@@ -56,7 +62,88 @@ If you have a Snyk token, add that as an environment variable:
 - Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables.
 - Set the variable name to _EXHORT_SNYK_TOKEN_, and copy-and-paste your Snyk token into the value field.
 
-#### Option I- As a build step
+
+#### General Configuration
+ Click <em>Manage Jenkins</em>. Click <em>System</em>, and scroll down to <em>Global properties/Environment Variables</em>. Here you can configure the following settings:
+ - name: _EXHORT_DEBUG_, Value: true , Description: Will invoke the analysis in verbose mode and will print a lot of useful logs to job output console - good for debugging, Default value is false.
+
+
+ - name: _EXHORT_DEV_MODE_, value: true, Description: Will invoke the Analysis on Staging Instance Of EXHORT Service, Default: false ( EXHORT Production Instance)
```

Comment:
For lines 68, and 71. We should put system-level variables, values, and commands in the mono-space font.

`_EXHORT_DEBUG_` --> \`EXHORT_DEBUG\`

`_EXHORT_DEV_MODE_` --> \`EXHORT_DEV_MODE\`

`true` --> \`true\`

`false` --> \`false\`

Reply:
Done, Thanks.